documentation/admonitions/_enable_pow.md

```diff
@@ -0,0 +1,7 @@
+:::note Enable Proof of Work (POW)
+
+This example requires an API endpoint that has "Proof of Work" enabled.
```

Comment:
Maybe we can remove this admonition because all examples have been updated to use local PoW. [PR](https://github.com/iotaledger/iota.js/pull/926)